### PR TITLE
Wiki categories

### PR DESCRIPTION
**Pull request by ablodge**
I'm satisfied that I have models without categories working.

### CODE REVIEW

File: wiki/urls.py [hunk 1]
```diff
@@ -132,7 +133,7 @@ def get_article_urls(self):
         urlpatterns = [
             # Paths decided by article_ids
             url(r'^(?P<article_id>\d+)/$',
-                self.article_view_class.as_view(),
+                cache_page(60 * 15)(self.article_view_class.as_view()),
```
Comment (nschneid) [inline wiki/urls.py]: Didn't you decide to remove the page caching?

File: wiki/plugins/metadata/forms.py [hunk 2]
```diff
@@ -82,6 +82,13 @@ def newArticle_ArticleCategory(self, name=None, parent=None, slug=None):
         newcategory.save()
         return newarticle, newcategory
 
+    def newArticle_without_category(self, name=None, parent=None, slug=None):
```
Comment (nschneid) [inline wiki/plugins/metadata/forms.py]: OK, now `Adposition`, `Construal`, `Usage`, and `Corpus` instances will not have an associated category.

**Comment by ablodge** [comment]
Let me know if this looks good.

**Comment by nschneid** [comment]
Yeah, that undoes the previous commit.

**Pull request merged**